account_reconcile_pos_payments/models/account_bank_statement.py

```diff
@@ -73,62 +72,73 @@ def _reconcile_pos(self):
+                    '%%%s%%' % rec.journal_id.cb_contract_number),
             ])
 
             i = 0
             for line in line_ids:
                 i += 1
-                _logger.debug('Processing line %s (%d/%d)' % (line.name, i, len(line_ids)))
+                _logger.debug('Processing line %s (%d/%d)' %
+                              (line.name, i, len(line_ids)))
 
                 # Try to find a CB statement older than our statement line
                 # but not older than 3 days before
-                limit_date = (
-                    datetime.strptime(line.date, '%Y-%m-%d')
-                    - timedelta(days=rec.journal_id.cb_delta_days))
+                limit_date = (datetime.strptime(
+                    str(line.date), '%Y-%m-%d') -
+                    timedelta(days=rec.journal_id.cb_delta_days))
                 cb_statement_ids = self.env['account.bank.statement'].search([
                     ('date', '<=', line.date),
                     ('date', '>=', limit_date.strftime('%Y-%m-%d')),
-                    ('balance_end_real', '>', round(line.amount-0.01, 2)),
-                    ('balance_end_real', '<', round(line.amount+0.01, 2)),
+                    ('balance_end_real', '>', round(line.amount - 0.01, 2)),
+                    ('balance_end_real', '<', round(line.amount + 0.01, 2)),
                     ('line_ids', '!=', False),
                 ])
 
-                _logger.debug('Possible CB statements: %d' % len(cb_statement_ids))
+                _logger.debug('Possible CB statements: %d' %
+                              len(cb_statement_ids))
 
-                # Only accept statements whose account movements are fully unreconciled
+                # Only accept statements whose
+                # account movements are fully unreconciled
                 ignore_cb_statement_ids = []
                 for cb_statement_id in cb_statement_ids:
                     reconciled_move_lines_count = \
                         len(cb_statement_id.move_line_ids.filtered(lambda l:
-                            l.reconciled and l.account_id.id == \
-                                rec.journal_id.default_debit_account_id.id))
+                            l.reconciled and l.account_id.id ==
+                            rec.journal_id.default_debit_account_id.id))
                     if reconciled_move_lines_count:
                         if len(cb_statement_id) != reconciled_move_lines_count:
-                            _logger.warning('Level 2: POS partially processed by the bank.')
+                            _logger.warning(
+                                'Level 2:POS partially processed by the bank.')
                         else:
-                            _logger.debug('Case 1: There are debits on the journal and they are reconciled.')
+                            _logger.debug('''Case 1: There are debits on
+                                the journal and they are reconciled.''')
```

Comment:
```suggestion
                            _logger.debug(
                                'Case 1: There are debits on '
                                'the journal and they are reconciled.')
```
minor comment: Usually this results in more readable code